### PR TITLE
Sigmund

### PR DESCRIPTION
**Pull request by sigmunsl**
Please review

### CODE REVIEW

File: content/about/employees/sigmund/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    src: sigmund.jpg
+email: sigmund@xal.no
+linkedin: https://www.linkedin.com/in/sigmund-slang-437321116/
+anchor: sigmund
+---
+
+Sigmund submitted his masters's degree in Geophysics and Seismology at 
+the University of Oslo in 2019. The subject of the thesis was Attenuation 
+of Seismic Interference Noise using Convolutional Neural Networks and was 
+written in collaboration with CGG. This work gave insight in the fields of
```
Comment (mandus) [inline content/about/employees/sigmund/index.md]: Is CGG an abbreviation, acronym, or something else (maybe it’s the full name of a company, and you may wwrite «with the company CGG» ?)

File: content/about/employees/sigmund/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+the University of Oslo in 2019. The subject of the thesis was Attenuation 
+of Seismic Interference Noise using Convolutional Neural Networks and was 
+written in collaboration with CGG. This work gave insight in the fields of 
+machine learning and seismic signal processing.
+
+<!--more-->
+
+Working with industrial problems utilizing machine learning has given him 
+experience with convolutional neural networks and seismic data processing  
+and further led to a consultancy job at Lundin Norway AS working with in
```
Comment (mandus) [inline content/about/employees/sigmund/index.md]: «with in related projects» sounds strange. «working on related projects» maybe?

File: content/about/employees/sigmund/index.md (hunk 1 above)
Comment (sigmunsl) [reply content/about/employees/sigmund/index.md]: CGG was a contractor company and is now a pure seismic processing company. I'll fix this.

File: content/about/employees/sigmund/index.md (hunk 2 above)
Comment (sigmunsl) [reply content/about/employees/sigmund/index.md]: That was a typo which I didn't catch.

**Pull request merged**